src/linker/Linker.Steps/SweepStep.cs

```diff
@@ -696,12 +711,17 @@ void UpdateTypeScope (Collection<ExportedType> forwarders)
 					if (td == null) {
 						// Forwarded type cannot be resolved but it was marked
 						// linker is running in --skip-unresolved true mode
+						var anr = (AssemblyNameReference) f.Scope;
+						f.Scope = importer.ImportReference (anr);
```

Comment:
Why is this needed?

Reply:
This was for consistency with how we treat unresolved typereferences here:
https://github.com/mono/linker/blob/73fbf96cb452f291cceba684034b8548d54d82eb/src/linker/Linker.Steps/SweepStep.cs#L908-L918

I think continue is right since we may be able to resolve other forwarders if not this one.